File: hack-2023/src/jsonObjects/JobLoadRequest.ts
import {MinMax} from "./MinMax";

export class JobLoadRequest{
    private hours:MinMax;
    private teamSize:MinMax;
    private allLocations:String[]

    constructor(hours:MinMax, teamSize:MinMax, allLocations:String[]) {
        this.hours = hours;
        this.teamSize = teamSize;
        this.allLocations = allLocations;
    }

}

File: backend/src/jsonObjects/JobLoadRequest.java
package jsonObjects;

public class JobLoadRequest {

    private MinMax hours;
    private MinMax teamSize;
    private Object[] allLocations;

    public JobLoadRequest(MinMax hours, MinMax teamSize, Object[] allLocations) {
        this.hours = hours;
        this.teamSize = teamSize;
        this.allLocations = allLocations;
    }

    public MinMax getHours() {
        return hours;
    }

    public void setHours(MinMax hours) {
        this.hours = hours;
    }

    public MinMax getTeamSize() {
        return teamSize;
    }

    public void setTeamSize(MinMax teamSize) {
        this.teamSize = teamSize;
    }

    public Object[] getAllLocations() {
        return allLocations;
    }

    public void setAllLocations(String[] allLocations) {
        this.allLocations = allLocations;
    }
}
